msg/AttentionRecognitionTopNItem.msg

```diff
@@ -0,0 +1,2 @@
+float64 value
```

Comment:
Agreed, will change to likelihood and add a comment noting that it is log_likelihood